mt940/models.py

```diff
@@ -381,7 +381,7 @@ def parse(self, data):
                     self.transactions.append(transaction)
                 else:
                     transaction.data.update(result)
-            elif tag.scope is Transaction and self.transactions:
```

Comment:
Can you explain the reasoning for this change? While it could still function with the current codebase this was written with custom inheritance in mind.